recipes/doccano/meta.yaml

```diff
@@ -0,0 +1,102 @@
+{% set version = "1.0.4" %}
+{% set node_version = os.environ.get('NODEJS_VERSION') or '13.6.0' %}
+
+package:
+  name: doccano
+  version: {{ version }}
+
+source:
+  url: https://github.com/doccano/doccano/archive/v{{ version }}.tar.gz
+  sha256: e74680d4bb677bf03bd27b4f3ead9fa84624bb85f7ab1063b11b90e923f65105
+  patches:
+    - rmv_drf_yasg.patch  # remove unneccesary dep drf_yasg in app/app/settings.py and app/app/urls.py
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - nodejs {{ node_version }}
+  host:
+    - pip
+    - python
+    - nodejs {{ node_version }}
+  run:
+    - python
+    - nodejs {{ node_version }}
+    - apache-libcloud >=2.4.0
+    - applicationinsights >=0.11.7
+    - colour >=0.1.5
+    - chardet >=3.0.4
+    - coverage >=4.5.3
+    - dj-database-url >=0.5.0
+    - django
+    - django-cloud-browser >=0.5.0
+    - django-cors-headers
+    - django-filter
+    - django-heroku >=0.3.1
+    - django-webpack-loader >=0.6.0
+    - django-widget-tweaks >=1.4.2
+    - django_polymorphic >=2.0.3
+    - django-pyodbc-azure >=2.1.0.0
+    - django-rest-polymorphic >=0.1.8
+    - djangorestframework >=3.10
+    - djangorestframework-csv >=2.1.0
+    - djangorestframework-filters >=0.10.2
+    - environs >=4.1.0
+    - djangorestframework-xml >=1.4.0
+    - faker >=0.9.1
+    - flake8 >=3.6.0
+    - furl >=2.0.0
+    - gunicorn >=19.9.0
+    - lockfile >=0.12.2
+    - model_bakery >=1.1.0
+    - mysqlclient >=1.4.2.post1
+    - psycopg2 >=2.7.7
+    - pyexcel >=0.5.14
+    - pyexcel-xlsx >=0.5.7
+    - python-dateutil >=2.7.3
+    - pytz >=2018.4
+    - requests >=2.21.0
+    - six >=1.11.0
+    - seqeval >=0.0.6
+    - social-auth-app-django >=3.1.0
+    - social-auth-core[azuread] >=3.0.0
+    - text-unidecode >=1.2
+    - unittest-xml-reporting >=2.5.1
+    - vcrpy >=2.0.1
+    - vcrpy-unittest >=0.1.7
+    - whitenoise[brotli] >=4.1.2
+    - conllu >=1.3.2
+
+test:
+  requires:
+    - python
+    - nodejs {{ node_version }}
+    - django
+
+  commands:
+    - cd app
+    - python3 manage.py migrate
+    - python3 manage.py create_admin --noinput --username "admin" --email "admin@example.com" --password "password"
+    - python3 manage.py test server.tests
+
+  source_files:
+    - app/
+
+about:
+  home: https://github.com/doccano/doccano
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
You also need to include the licenses of the bundled `npm` packages.

Reply:
What's the preferred method of adding multiple licenses to a recipe?

Reply:
Get rid of `license_family`, use a SPDX license expression and provide a list of license files, i.e.
```yaml
  license: MIT AND LGPL-2.0-OR-LATER
  license_file:
    - LICENSE
    - OTHER_LICENSE
```
You can also use things like `MIT AND (LGPL-2.0-or-later OR Apache-2.0) AND BSD-3-Clause`, see https://spdx.org/licenses/

Reply:
@xhochy After looking at the doccano package json I'm having trouble determining which packages are bundled? Also doesn't look like any of the npm installed packages on conda-forge have multiple licenses in their recipes

Reply:
@xhochy bump

Reply:
Passing the review to someone else on that, for `code-server` we actually bundle all licenses but the bundling is done by VSCode: https://github.com/conda-forge/code-server-feedstock/blob/458f7f4d19b0682aaf821177efc210ccbb870169/recipe/meta.yaml#L36

Reply:
@conda-forge/core can someone familiar with win take a look at this?